vault/resource_database_secret_backend_static_role_test.go

```diff
@@ -155,7 +207,65 @@ func createTestUser(connURL, username string) error {
 	return nil
 }
 
-func testAccDatabaseSecretBackendStaticRoleConfig_basic(name, username, db, path, connURL string) string {
+func testAccDatabaseSecretBackendStaticRoleConfig_rotationSchedule(name, username, db, path, connURL string) string {
+	return fmt.Sprintf(`
+resource "vault_mount" "db" {
+  path = "%s"
+  type = "database"
+}
+
+resource "vault_database_secret_backend_connection" "test" {
+  backend = vault_mount.db.path
+  name = "%s"
+  allowed_roles = ["*"]
+
+  mysql {
+	  connection_url = "%s"
+  }
+}
+
+resource "vault_database_secret_backend_static_role" "test" {
+  backend = vault_mount.db.path
+  db_name = vault_database_secret_backend_connection.test.name
+  name = "%s"
+  username = "%s"
+  rotation_schedule = "* * * * *"
+  rotation_window = 3600
+  rotation_statements = ["ALTER USER '{{username}}'@'localhost' IDENTIFIED BY '{{password}}';"]
+}
+`, path, db, connURL, name, username)
+}
+
+func testAccDatabaseSecretBackendStaticRoleConfig_updatedRotationSchedule(name, username, db, path, connURL string) string {
```

Comment:
Not entirely sure yet why this test is failing now and not updating a name. Haven't really changed anything that would cause this in this test I don't think? 🤔

Reply:
Test is failing with

> cannot update static account username

which is expected if the test is updating the name